src/web/components/MostViewed/MostViewedRight/MostViewedRightWrapper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { css } from 'emotion';
+import { MostViewedRight } from './MostViewedRight';
+
+const flexGrow = css`
+    flex-grow: 1;
+`;
+
+interface Props {
+    limitItems?: number;
+}
+
+// Wrapping MostViewedRight so we can determine whether or not there's enough vertical space in the container to render it.
+export const MostViewedRightWrapper = ({ ...props }: Props) => {
```

Comment:
Could this be changed to:

```suggestion
export const MostViewedRightWrapper = (props: Props) => {
```

I.e. do we need the destructuring etc?

Reply:
Yeah good spot - updated